test/unit/test.database.js

```diff
@@ -1,13 +1,247 @@
 const environments = require("../../knexfile.js");
 const Knex = require("knex");
 const { postProcessDatabaseResults } = require("src/server/post-process-database-results");
+const { processBatchOfLogs } = require("src/blockchain/download-augur-logs");
+const Augur = require("augur.js");
+const uuid = require("uuid");
+const deepmerge = require("deepmerge");
 
-module.exports = () => {
-  const env = Object.assign({}, environments.test, {
+
+function getEnv() {
+  return Object.assign({}, environments.test, {
     postProcessResponse: postProcessDatabaseResults,
   });
+}
+
+function contains(container, element) {
```

Comment:
augur-node already pays the price of lodash for these sorts of functions, this one could be replaced by `_.includes`

Reply:
Oh perfect I'll switch to that.